gapps_users_to_contacts_group_copier/gapps_users_to_contacts_group_copier.py

```diff
@@ -4,12 +4,13 @@
 __author__ = "vsin"
 __version__ = "1.0.0"
```

Comment:
Forgot to ask about this!
1) Extend to __authors__ = [names] in each script
2) Attribute to Futurice, and can keep a separate contributors file?